src/kanata/mod.rs

```diff
@@ -556,6 +559,13 @@ impl Kanata {
                 } else {
                     mmsv.distance = mmas.max_distance;
                 }
+                for &modifier in &self.move_mouse_speed_modifiers {
+                    mmsv.distance = u16::max(
+                        (mmsv.distance as f32 * (modifier as f32 / 100 as f32)).round() as u16,
```

Comment:
I think we should use https://doc.rust-lang.org/std/primitive.f32.html#method.min here and in the other multiplication too
e.g. `f32::min(..., f32::from(u16::MAX))`

Reply:
Another thought: this seems like a good candidate for splitting into a function that can have unit tests.

Reply:
Where do you think the test should go? I couldn't find any similar ones. Should I put it at the bottom of `src/kanata/mod.rs` along with the function itself?

Reply:
Yea that'll do 👍